jekyll/_cci2/server/Installation/phase-1-prerequisites.adoc

```diff
@@ -247,7 +252,7 @@ ifndef::env-gcp[]
 . If you are using AWS Route 53 for DNS, you need the *certbot-route53* plugin installed. You can install the plugin with the following command:
 +
 ```shell
-pip3 install certbot-dns-route53
+pip install certbot-dns-route53
```

Comment:
I am not sure this is right, but i could not get `pip3` to work.  Please help me if you know the answer

Reply:
that works, thanks